### PR TITLE
[imager] Add restore cmd to expose cleanup workflow

### PR DESCRIPTION
**Pull request by leo8a**
This exposes the cleanup workflow via a new restore cmd. Particularly handy when re-running such workflow is needed (e.g., after workflow failure).

Related to #79.

### CODE REVIEW

File: ibu-imager/cmd/restore.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+func init() {
+
+	// Add restore command
+	rootCmd.AddCommand(restoreCmd)
+
+	// Add flags to restore command
+	restoreCmd.Flags().StringVarP(&authFile, "authfile", "a", common.ImageRegistryAuthFile, "The path to the authentication file of the container registry.")
+	restoreCmd.Flags().StringVarP(&containerRegistry, "image", "i", "", "The full image name with the container registry to push the OCI image.")
+	restoreCmd.Flags().StringVarP(&recertContainerImage, "recert-image", "e", common.DefaultRecertImage, "The full image name for the recert container tool.")
+	restoreCmd.Flags().BoolVarP(&recertSkipValidation, "skip-recert-validation", "", false, "Skips the validations performed by the recert tool.")
```
[the remark this answers is not in the document]
Comment (leo8a) [reply ibu-imager/cmd/restore.go]: +1 thanks

**Review by donpenney — COMMENTED** [review]
/lgtm

**Pull request merged**